### PR TITLE
source system profile

### PR DESCRIPTION
**Pull request by southeo**
Adds request and profile for source system pid records

note: If we use “allof”, we can’t generate code using our code gen library. Made a jira ticket to address other schemas to align them with this one

### CODE REVIEW

File: data-model/fdo-profiles/0.1.0/source-system/schema/source-system-request.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "description": "Kernel from which all DiSSCo FAIR Digital Object profiles are derived",
+  "$comment":"FDO Profile Version 0.1.0",
+  "properties": {
+    "fdoProfile" : {
+      "type": "string",
+      "description": "PID to a machine readable description of the attributes in the FDO record",
+      "example": "https://hdl.handle.net/21.T11148/d8de0819e144e4096645",
+      "$comment": "idx = 1"
+    },
+    "digitalObjectType": {
```
Comment (TomDijkema) [inline data-model/fdo-profiles/0.1.0/source-system/schema/source-system-request.json:14]: Shouldn't this be fdoType?

**Review by TomDijkema — COMMENTED** [review]
I am missing a bit of context, are you updating the source system profile or this a completely new schema for the fdo of a source system?

File: data-model/fdo-profiles/0.1.0/source-system/schema/source-system-request.json (hunk 1 above)
Comment (southeo) [reply data-model/fdo-profiles/0.1.0/source-system/schema/source-system-request.json:14]: No, this is correct. It's based on the handle profile, which calls this field "digitalObjectType"

**Review by TomDijkema — APPROVED** [review]
Not really into these fdo profiles, but this looks ok!

**Pull request merged**